schedule/disperse_siblings.py

```diff
@@ -47,13 +47,13 @@ def get_due_range(cid, parameters, stability, due):
     revlogs = filter_revlogs(mw.col.card_stats_data(cid).revlog)
     last_due = get_last_review_date(revlogs[0])
     last_rating = revlogs[0].button_chosen
-    if last_rating == 4:
-        try:
-            new_ivl = int(round(stability * parameters['e'] * math.log(parameters['r']) / math.log(0.9)))
-        except KeyError:
-            new_ivl = int(round(9 * stability * (1 / parameters['r'] - 1)))
+    if version[0] == 4:
```

Comment:
We need pass the version variable when calling this function.

Reply:
Version is not a global variable.

Reply:
So, how to do it?

Reply:
Have you tested it in your device? I will review it tomorrow.

Reply:
I have not tested it because I don't exactly how to test this.

Reply:
You can replace the code in your local add-on file.

Reply:
> You can replace the code in your local add-on file.

I knew that. The problem was that I didn't know what to look while testing. But, then I decided to test it anyway. I initially got an error and then fixed it. Apart from that, it seems to work fine (but I am not sure).